File: csharp/5.cs
public class Solution {

  static void Main(string[] args) {
    Console.WriteLine("\n\n\n" + IsPalindrome(121));
    Console.WriteLine(IsPalindrome(212));
    Console.WriteLine(IsPalindrome(98) + "\n\n\n");
    
    Console.Read();
  }

  public static bool IsPalindrome(int x) {
    string numberString = x.ToString();
    string reversedString = new string(numberString.Reverse().ToArray());
    return numberString == reversedString;
  }
}

File: rust/5.rs
fn is_palindrome(x: i32) -> bool {
   let number_string = x.to_string();
   let reversed_string: String = number_string.chars().rev().collect();
   number_string == reversed_string
}

fn main() {
   println!("\n\n\n{}", is_palindrome(121));
   println!("{}", is_palindrome(212));
   println!("{}", is_palindrome(98));
}
 